lms/templates/discussion/index.html

```diff
@@ -22,6 +22,13 @@
 ## Enable fast preview to fix discussion MathJax rendering bug when page first loads.
 <%include file="_js_body_dependencies.html" args="disable_fast_preview=False"/>
 <%static:js group='discussion'/>
+<script language="javascript">
+   $(function () {
+       $("section.discussion").each(function (index, elem) {
+           DiscussionApp.start(elem, true);
+       });
+   });
+ </script>
```

Comment:
@bradenmacdonald mostly to reduce code drift - but we need it in solutions so that the 3rd party tool that should not be named could work; can't recall the details, but probably jquery.xblock had some trouble with this.